workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/Utility/EntityBuilderTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Improbable.Worker.Core;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core.EditmodeTests
+{
+    [TestFixture]
+    public class EntityBuilderTest
+    {
+        [Test]
+        public void EntityBuilder_should_throw_exception_if_no_position_component_is_added()
+        {
+            var builder = EntityBuilder.Begin();
+            Assert.Throws<InvalidEntityException>(() => builder.Build());
```

Comment:
The test reveals an opportunity to fix the error message:

`"Entity is invalid. Missing one of required component: Position and EntityAcl"`

Reply:
Ah yeah - that was from before I added the auto-building ACL. Will edit the message